build.sbt

```diff
@@ -0,0 +1,21 @@
+val TwoTenScalaVersion = "2.10.5"
+val TwoElevenScalaVersion = "2.11.11"
+
+organization := "com.mguaypaq.spark"
```

Comment:
I think this should be `"ca.pointedset"`; I'll add an extra commit in this PR to fix it.